RestructuredText/reStructuredText.sublime-syntax

```diff
@@ -77,6 +88,17 @@ contexts:
       scope: markup.underline.substitution.restructuredtext
       captures:
         1: punctuation.definition.substitution.restructuredtext
+    - match: (:math:)(`)
```

Comment:
Only the `math` part should become `support.function`, not the punctuation.

Reply:
I'm not sure I agree, based on how github highlights the word `:test:` in the rst file. If nothing else, I find it more helpful as a reader to see `:math:` in a single color. Ok to be overruled if you think being precise is more important than looking pretty

Reply:
I tried to think of similar applications and apparently we've been doing things in either of two ways:

1. Scope everything, including punctuation, as the keyword. An example are at-rules in CSS, i.e. `@media`, or dollar signs for languages that *require* them for variables, like Php with `$var`.
2. Only scope the name, not the punctuoation. Examples for this are annotations in Java or decorators in Python, but those are different because the punctuation there doesn't describe an atomic but rather a construct, which is why it also gets a meta scope.

This case fits into 1., so I we probably should include the colons at least until we decided on a uniform standard for this type of token.
Either way, punctuation *should* receive a `punctuation` scope, either on top of the `support.function` scope or without it entirely.